swift-sdk/Internal/InboxImpressionTracker.swift

```diff
@@ -81,13 +93,13 @@ class InboxImpressionTracker {
 }
 
 extension InboxImpressionTracker.RowInfo: Hashable {
```

Comment:
You can make this public instead. I don't think you need to mark every method in protocol as public if you mark the extension as public.

Reply:
```suggestion
public extension InboxImpressionTracker.RowInfo: Hashable {
```